src/cascadia/TerminalApp/Pane.cpp

```diff
@@ -646,13 +720,13 @@ std::pair<Pane::PanePoint, Pane::PanePoint> Pane::_GetOffsetsForPane(const PaneP
 
     if (_splitState == SplitState::Horizontal)
     {
-        secondOffset.y += (1 - _desiredSplitPosition) * parentOffset.scaleY;
+        secondOffset.y +=  _desiredSplitPosition * parentOffset.scaleY;

```

Comment:
So, this is a bug in preview that directional movement will be broken during startup if you use splits that aren't 50%, but amplified with https://github.com/microsoft/terminal/pull/11046 by removing the actual size logic.